_posts/2019-04-14-fastqc-long-reads.md

```diff
@@ -0,0 +1,24 @@
+---
+site: freiburg
+tags: [devops, tools]
+title: FASTQC jobs and very long reads
+location: Galaxy Europe
+---
+
+Over the last weeks we saw many [FASTQC]({{ page.website }}/?tool_id=toolshed.g2.bx.psu.edu/repos/devteam/fastqc/fastqc/0.72) jobs running for days or even longer, which is not normal.
+We investigated this and saw that all of those jobs are processing files with unusually long reads.
+
+We are excited to see that our instance is used more and more for processing data of the
+fourth-generation of DNA sequencing technology!
+
+However, FASTQC seems to have a problem with that so we updated the tool and tried to understand what is going on.
+We finally found that FASTQC has a hard-coded parameter (-Xmx250m) to restrict the Java virtual machine to a certain amount of memory.
+Increasing this value made all our tests way more faster with very long reads and we hope to speed up your computation from days to minutes.
```

Comment:
all our tests → all of our tests
more faster with very long reads → much faster for very long reads